FreeRTOS/FreeRTOS-Kernel
# Convert all files in this repository to LF (unix) line endings.

Description
-----------
Convert all files in this repository to LF (unix) line endings.
Set text=auto for all files to ensure that files have correct line endings upon checkin.

Refer to the [github docs on line-endings](https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings) for more information.

In general, git handles LF line endings in a much more sane way than it does CRLF line endings. Users of windows should see little change to the way the use this repository. By default, git with automatically convert to/from native line endings during checkout and commit operations.

For more info on these settings in git, check out the [git documentation](https://git-scm.com/docs/gitattributes#_text) on the subject.

Test Steps
-----------
Validated that new clones of the repository handle automatic line ending conversion correctly on windows and *nix.

Related Issue
-----------
#234 #340 #339 #247


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
